benchmarks/README.md

```diff
@@ -0,0 +1,68 @@
+# Benchmark runner app for Datadog React Native SDK
+
+# Build and run
+
+- Run yarn install from root folder
+- From iOS Folder run pod install
+- Run yarn ios/android from root folder
+
+## Enabling New architcture
+
+### iOS
+- From ios folder, run:
+RCT_NEW_ARCH_ENABLED=1 pod install
+
+### Android
+Set newArchEnabled=true on benchmarks/android/gradle.properties
+
+# ENV config
+
+The .env config file holds the Datadog API config, and also allows to set up a test scenario that will be launched directly when the app boots up. If no scenario is provided the app will start without loading any scenario and await for a deeplink to trigger a test scenario.
+
+    DD_CLIENT_TOKEN="CLIENT_TOKEN"
+    DD_API_KEY="API_KEY"
+    DD_APP_ID="APP_ID"
+    DD_SITE="SITE"
+    DD_ENV="ENV"
+    BENCH_SCENARIO=""
+    BENCH_RUN_TYPE=""
+
+# Changing ENV
+
+### Debug
+     $ ENVFILE=.env.alternate [yarn ios/yarn android]
+
+### Release 
+    $ ENVFILE=.env.alternate ./gradlew assembleRelease
+    iOS - create schemes and select them during xcodebuild
+
+# Setting test scenarios via Deeplink
+
+Test scenarios can be triggered by running a deeplink with a specific set of parameters to the benchmark app.
+
+## Run Test Scenario
+
+Launches a specific test scenario with a particular configuration.
+
+- **Method**: start
+- **Parameters**:
+    - **scenario**: name of the test as defined on Scenario enum in types/testConfig.ts
+    - **runType**: `instumented|baseline|profiling`
```

Comment:
This feels like it could benefit from an example!
```suggestion
Launch a specific test scenario with a particular configuration:

- **Method**: `start`
- **Parameters**:
    - **scenario**: name of the test as defined on Scenario enum in `types/testConfig.ts`
    - **runType**: `instrumented|baseline|profiling`
```